client/extensions/wp-super-cache/wrap-settings-form.jsx

```diff
@@ -2,14 +2,18 @@
  * External dependencies
  */
 import React, { Component } from 'react';
-import { flowRight } from 'lodash';
+import { flowRight, isEqual, omit, pick } from 'lodash';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import { protectForm } from 'lib/protect-form';
 import trackForm from 'lib/track-form';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSettings } from './state/selectors';
```

Comment:
I think that because we're already in the `extensions/wp-super-cache` folder there's little chance that there would be confusion with Calypso's settings for example. Is that what you mean?